src/editor/data/templates/companies-block.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
We have different between mock data and templates ? If not, maybe we need to use mock data immediately ?

Reply:
I think we will join templates data and data from stories, but later, it requires some actions

Reply:
> We have different between mock data and templates ? If not, maybe we need to use mock data immediately ?

I think we will join templates data and data from stories, but later, it requires some actions